web/src/modules/layout/components/__tests__/Footer.spec.js

```diff
@@ -6,20 +6,20 @@ import { shallow } from 'enzyme'
 
 describe('Footer', () => {
   it('should insert items', () => {
+    const preVersion = global.__VERSION__
+    global.__VERSION__ = 'vX.X'
     const component = <Footer>
       <div>MockNode</div>
       <div>AnotherMockNode</div>
     </Footer>
     expect(shallow(component)).toMatchSnapshot()
+    global.__VERSION__ = preVersion
```

Comment:
I think you can remove resetting the version here. Afaik each test suite has it own environment instance (and is therefore only set for this File. See [here](https://jestjs.io/docs/en/configuration#globals-object)

Reply:
I think it is good practice to clean up after setting something global. This could easily affect tests in other tests.

Reply:
Agreeing with @maxammann. In this case it is not necessary, but won't hurt and potentially prevent wrong behavior/bugs.